src/pages/cookie-settings.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Cookie Settings"
+date: "2022-01-25"
+---
+
+import ManageCookieSettings from '@site/src/components/ManageCookieSettings'
+
+# Cookie settings
+
+### About this cookie policy
+
+This Cookie Policy explains what cookies are and how we use them. You should read this policy to understand what cookies are, how we use them, the types of cookies we use i.e, the information we collect using cookies and how that information is used and how to control the cookie preferences. For further information on how we use, store and keep your personal data secure, see our Privacy Policy.
+
+You can at any time change or withdraw your consent from the Cookie Declaration on our website. Learn more about who we are, how you can contact us and how we process personal data in our Privacy Policy. Your consent applies to the following domains: docs.snowplowanalytics.com
+
+:::note
```

Comment:
I would move this down to “How can I control cookie preferences”

Reply:
I think it makes sense to keep in "above the fold" (visible on screen by default) as it is in the current docs. Otherwise I think nobody would bother to scroll down to find it. (Assuming we want to optimize for people to use this - if that's not the case its a good idea to move it down)